pkgs/data/fonts/iosevka/bin.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchzip }:
 
 let
-  version = "3.4.6";
+  version = "3.6.2";
```

Comment:
Do `iosevka` and `iosevka-bin` always have to be in sync with each other? If so, I think we should ensure they'd always stay the same by changing this line to `version = iosevka.version;`.

Reply:
They don't need to be, it just happens that `iosevka-bin` gets updated automatically, while `iosevka` needs the `package.json` to be regenerated. It'd make it less confusing if they were updated in lockstep though.